src/main/java/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;
+
+import model.FruitTransaction;
+import service.FruitTransactionParser;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    @Override
+    public FruitTransaction parse(String line) {

```

Comment:
It's good practice to validate the input `line` for null and empty values before processing it to avoid potential `NullPointerException` or unexpected behavior.